pom.xml

```diff
@@ -689,6 +689,22 @@
                     <version>1.2.1</version>
                 </plugin>
                 <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <version>3.1</version>
```

Comment:
no need to specify the version, we already have a more recent version of maven-compiler-plugin as part of the parent pom